### ISSUE TITLE
Virtual batch Normalization

### ISSUE THREAD

**Issue by sahiliitm** [openai/evolution-strategies-starter#8]
If I understand the code correctly, it uses virtual batch normalization only for the inputs and *not* for the intermediate layers.

Was this done in the Atari context for getting the results stated in the paper?

**Comment by joyousrabbit** [comment]
@sahiliitm Hello, in the paper, it is stated " We used the same preprocessing and feedforward CNN architecture used by (Mnih et al., 2016)". So it should be the traditional two layers FF.

**Comment by PatrykChrabaszcz** [comment]
Hi @sahiliitm 

Could you point me to the code where you see virtual batch normalisation implementation?
I thought it is:

```python
**@property
def needs_ref_batch(self):
     return False**
```
Which is currently not implemented

**Comment by louiskirsch** [comment]
In the code here we have just z-normalization for the inputs, no virtual batch norm.
Also no hyperparameters for Atari. OpenAI, please be more reproducible! :-)
